README.md

```diff
@@ -1,6 +1,5 @@
 # bitcoinjs-message
 
-[![TRAVIS](https://secure.travis-ci.org/bitcoinjs/bitcoin-message.png)](http://travis-ci.org/bitcoinjs/bitcoin-message)
```

Comment:
Why remove TRAVIS?
It seems the url is wrong, should be `/bitcoinjs-message` instead.